src/Microsoft.Health.Fhir.Core/Exceptions/InitializationException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.Core.Exceptions
+{
+    public class InitializationException : Exception
```

Comment:
Should this inherit from FhirException?

Reply:
I don't think so because it isn't related to FHIR, it is related to the server startup.